tslint.json

```diff
@@ -1,73 +1,64 @@
 {
```

Comment:
😕 did Prettier automatically apply these changes for you? If so, that's ok, but if not, could you please revert the unrelated lines?

Reply:
Yes, Prettier has formatted the file during the precommit.
I only added the following line:
```typescript
"max-nesting-depth": [false],
```

Reply:
Heh. I thought I'd gotten all of them in #4270, looks like I missed this. Thanks!